TWLight/users/views.py

```diff
@@ -848,3 +848,15 @@ def form_valid(self, form):
 class LibraryRedirectView(RedirectView):
     permanent = True
     url = reverse_lazy("users:my_library")
+
+
+class DeleteApplication(RedirectView):
```

Comment:
Instead of `DeleteApplication`, can you change the class name to `WithdrawApplication` since we do not want to delete the application?